From 24bce7abc0045474b9e6873952d95b24a2c934f8 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:22:25 -0400
Subject: [redhat] kabi: Adding symbol aio_complete (fs/aio.c)

Message-id: <53f729ce7339bc895cefe4d1fed57c2ee207ca99.1410510773.git.poros@redhat.com>
Patchwork-id: 95981
O-Subject: [PATCH RHEL7.1 025/233] kabi: Adding symbol aio_complete (fs/aio.c)
Bugzilla: 1071097
RH-Acked-by: Jeff Moyer <jmoyer@redhat.com>

Bugzilla: 1071097
    https://bugzilla.redhat.com/show_bug.cgi?id=1071097

    Description:
    ============
    Notify the AIO subsystem on completion of IO.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/aio_complete
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/aio_complete
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/aio_complete

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/aio_complete  b/redhat/kabi/kabi-module/kabi_ppc64/aio_complete 
new file mode 100644
index 0000000..1d6fc2d
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/aio_complete  
@@ -0,0 +1,2 @@
+#1-
+0x74f612e9 aio_complete vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/aio_complete  b/redhat/kabi/kabi-module/kabi_s390x/aio_complete 
new file mode 100644
index 0000000..ee7ed9a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/aio_complete  
@@ -0,0 +1,2 @@
+#1-
+0x18feadd3 aio_complete vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/aio_complete  b/redhat/kabi/kabi-module/kabi_x86_64/aio_complete 
new file mode 100644
index 0000000..32eba03
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/aio_complete  
@@ -0,0 +1,2 @@
+#1-
+0xf713bbef aio_complete vmlinux EXPORT_SYMBOL
-- 
1.7.1